src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+---
+title: Issues API using Nerdgraph
+metaDescription: "Set capabilities and entitlements in Detection and AI"
+redirects: 
+---
+
+This doc is a place for you and your team to learn how to query your data for issues and incidents and then configure features using our NerdGraph API.
```

Comment:
I think we need more context here. 'issues' and 'incidents' are pretty general words that could apply to other features. I'd go into more detail about what the scope of this is for and link to relevant regular NR docs that relate. For example, something like 'For our alerts and applied intelligence features, we have issues and incidents, which are...[links to relevant stuff to learn more] You can configure them in the UI, and you can also use our NerdGraph API to do this..."

Reply:
@neta-signifai would you mind providing a little more context about this Nerdgraph tutorial? I want to make sure I have a handle on the scope of this document.